### ISSUE TITLE
Confusing documentation: exports.usage()

### ISSUE THREAD

**Issue by aubergine10** [totaljs/framework#389]
Just trying out totaljs for first time and I'm a bit confused about the exports.usage() in a model.

The [documentation](http://docs.totaljs.com/v1.9.x/en.html#pages~Models) states:

> This function must return exploitation of the module.

Might be translation issue, but I'm assuming that by "exploitation" you mean "usage"? If so, I still don't know what "usage" really means. Could you elaborate?

Also, I'm finding the documentation quite hard to follow in general - perhaps I'm missing a lot of prerequisite knowledge relating to Node.js or something. Is there somewhere I can send pull requests to help improve the documentation?

**Comment by petersirka** [comment]
HI @aubergine10
yes, I have a lot of problems in documentation with grammar. I'll be really happy when you help me with it.

I prepare a new documentation for total.js v2.0.0 <https://github.com/totaljs/documentation> so you can edit this branch (I'll be happy).

__File:__
<https://github.com/totaljs/documentation/tree/master/public/v2.0.x/en.json>

Please commit each change because I have problem with merging file `en.json` (it's too big) when is a change. Thank you !!!!!! 👍  💯  👍

**Comment by petersirka** [comment]
I appreciate it 👍

**Comment by aubergine10** [comment]
Ok, will do :)

But first... what does the exports.usage() method do? I see it in many examples, but I don't know what's calling the method or why the method is returning what often looks like internal variables (I assume each module is sandboxed in to it's own scope?)

**Comment by petersirka** [comment]
`exports.usage()` returns information about usage of `model`, `module`, `controller`. The method `F.usage()` collects all information from all dependencies.

Output from the method `F.usage()` (serialized into the JSON):
<http://www.gpslocalization.com/usage/>

The picture below shows all registered modules, try to find `reqstats` module and its `usage` is null because the module doesn't contain `exports.usage`.

![image](https://cloud.githubusercontent.com/assets/2414252/14831599/b89dbcd0-0bf6-11e6-95fa-7e8f8109534d.png)

Do you understand?
Thank you!

**Comment by aubergine10** [comment]
Ah, so it's a way to find out what other parts of an app are using a specific module? And with that you can tell what will be affected should the module change, etc?

**Comment by petersirka** [comment]
Yes and no. `usage` offers only information about module/model/controller usage (nothing more).

**Comment by aubergine10** [comment]
So the exports.usage() can return anything the developer chooses (as long as it's an object?) - is it mainly a debug thing for quick summary on what's going on in a module?

**Comment by petersirka** [comment]
Yes 💯

**Comment by aubergine10** [comment]
I'm looking at the en.json file now... It really is huge!

I was wondering if it would be better to have one .json file acting as an index, and then separate .json files for each property/method/event/etc... That would make it much easier to edit and also track changes more easily. There would be lots more files, but overall I think it might make the localisation easier to maintain.

**Comment by petersirka** [comment]
I understand but I think that it's not a good idea because documentation is offline and I'll minify the `en.json` file. I don't have any problem with editing the file, a good tool for editing JSON <https://tickplant.com/powerjsoneditor/> and Sublime Text with String utilities plugin, it supports `json escape` and `json unescape`. I don't want to change a structure of the docs, it's not perfect but it works 👍 
---
I need improve grammar. Thank you

**Comment by petersirka** [comment]
Thank you!
